counterparty-rs/src/utils.rs

```diff
@@ -40,10 +40,14 @@ fn script_to_address(script_pubkey: Vec<u8>, network: &str) -> PyResult<String>
             Err(_) => return Err(PyErr::new::<pyo3::exceptions::PyValueError, _>("Invalid version value")),
         };
 
+        let n = 22;
+        if script.len() < n{
```

Comment:
hum.. should not be here 24 ? because we take script[2..22] ?

Reply:
@ouziel-slama 22 is an index not the number of bytes to take: https://doc.rust-lang.org/std/ops/struct.Range.html. We're taking 20 bytes here, from the byte at index 2 up to the byte at index 22 (exclusive). We skipped 2 bytes so the length of the script must at least be 22.

Reply:
ah right!! I forgot that you should never do a review before the second coffee lol!